artifactory/utils/container/image.go

```diff
@@ -152,8 +152,12 @@ func (image *Image) GetRemoteRepo(serviceManager artifactory.ArtifactoryServices
 	if err != nil {
 		return "", err
 	}
+	errorMessage := "error while getting docker repository name. Artifactory response: " + resp.Status + "."
```

Comment:
Check the format of error. is it spanning into multiple lines?
```suggestion
	if statusOk := resp.StatusCode == http.StatusOK; !statusOk && resp.StatusCode == http.StatusForbidden {
		return "", errorutils.CheckErrorf(getStatusForbiddenErrorMessage())
	} else if !statusOk {
		return "", errorutils.CheckErrorf("error while getting docker repository name. Artifactory response: " + resp.Status)
	}
```
```
func getStatusForbiddenErrorMessage() string {
	return fmt.Sprintf(`error while getting docker repository name. Artifactory response: %d
	Possible causes include Xray scan in progress, Xray policy violations, insufficient permissions, invalid authentication method, disabled anonymous access, missing Docker manifests.
	Please verify the above factors to resolve the issue.`, http.StatusForbidden)
}
```